packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsWithAuthTest.e2e.test.ts

```diff
@@ -25,6 +25,9 @@ import {
   configureAmplify,
 } from '../cognitoUtils';
 import 'isomorphic-fetch';
+import { API } from 'aws-amplify';
+import { GRAPHQL_AUTH_MODE } from '@aws-amplify/api';
```

Comment:
Can we update the `package.json` to get this dependency instead of getting it transitively thorough `aws-amplify`